### ISSUE TITLE
Delete account for Didentity frontend

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#16]
## User story
1. As a {HR-Chief}
2. I want / need {to be able to delete frontend user accounts of HR-employees }
3. So that {no outdated accounts are left in the system forever}

## Acceptance criteria
* Accounts can only be deleted by admin-role
* a logfile should contain information about name of deleted account and time of deletion
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
